app/services/plan_service/store/plan.rb

```diff
@@ -2,20 +2,26 @@ module PlanService::Store::Plan
 
   class TrialModel < ActiveRecord::Base
     self.table_name = :marketplace_trials
+    serialize :features, Array
```

Comment:
@rap1ds sorry, misinterpreted your intention. Yes it does make sense not to store that information.